Recruit-Training/Advanced-Recruit-Training/src/BatteryStateOfChargeService.h

```diff
@@ -27,6 +27,13 @@ class BatteryStateOfChargeService : public I_BatteryStateOfChargeService
 private:
     // Convention is to have an _ after private members
     const double initialStateOfChargePercent_;
+    double old_current;
+    double present_batterycurrent;
+    double totalAmpHoursUsed_;
+    bool isCharging_;
+    QTime timeWhenChargedOrDepleted_;
+     QTime old_time;
+    QTime present_time;
```

Comment:
Solar car standards are to use Camel Case, not Snake Case. All private variables should end with an underscore. Ex: old_current -> oldCurrent_.